frontend/views/components/AvatarGenerator.vue

```diff
@@ -0,0 +1,48 @@
+<template lang='pug'>
+  // Leave in the DOM for debug
+  div(style='display: none;')
+    img(:src='ephemeral.avatarBase64')
```

Comment:
It doesn't make sense to me why this code is inside of a Vue Component that does nothing to display anything. If the entire purpose of this component is to call `blockies.create`, then this would be better implemented as a function, and called via `sbp` inside of SignupForm.

Reply:
Yes, it could be adapted to a normal function, but I found it very useful to be inside a Vue component to be more "vue-sh format". I also find it very handy to be the image in the DOM for debug (or even tests) purposes. I'll leave it as it is to be quicker, feel free to change it in a next MR